src/common/options/mds-client.yaml.in

```diff
@@ -390,6 +390,22 @@ options:
   default: true
   services:
   - mds_client
+- name: fuse_entry_timeout
+  type: int
+  level: advanced
+  desc: timeout for entry in kernel
+  long_desc: in particular, the fuse_entry_timeout is designed as a member of the client class.
```

Comment:
```
  90         /** Validity timeout (in seconds) for inode attributes. If
  91             attributes only change as a result of requests that come
  92             through the kernel, this should be set to a very large
  93             value. */
  94         double attr_timeout;
  95   
  96         /** Validity timeout (in seconds) for the name. If directory
  97             entries are changed/deleted only as a result of requests
  98             that come through the kernel, this should be set to a very
  99             large value. */
 100         double entry_timeout;
```

The above is copied from the libfuse repo. Please fix your `desc` and `long_desc` with this.